torchbox/torchbox.com
# Add extra alt text options to the work page image

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1579462155)

### Description of Changes Made

- Add extra fields for the work page image to allow custom alt text on the page, or the ability to mark the image as decorative with empty alt text.

### How to Test

- Edit a work page on your local build and try setting custom alt text, and marking the image as decorative. Note that it both are done at the same time, the image will be marked as decorative and the custom alt text ignored.

### Screenshots

<details>
  <summary>Expand to see more</summary>

![Screenshot 2024-07-30 at 11 37 23](https://github.com/user-attachments/assets/b1297cf9-d399-469b-bcde-45df69e67862)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
